lib/publiccloud/gcp_client.pm

```diff
@@ -33,13 +33,9 @@ sub init {
     $self->project_id($data->{project_id});
     $self->account($data->{client_id});
     assert_script_run('source ~/.bashrc');
-    if (is_sle('>=15')) {
-        # kill it in case it's running
-        script_run("killall chronyd && sleep 5 && if pgrep chronyd; then killall -9 chronyd; fi");
-        assert_script_run("chronyd -q 'pool time.google.com iburst'");
-    } else {
-        assert_script_run('ntpdate -s time.google.com');
-    }
+    # kill it in case it's running
+    systemctl('stop chronyd.service');
+    assert_script_run("chronyd -q 'pool time.google.com iburst'");
```

Comment:
```suggestion
    script_retry("chronyd -q 'pool time.google.com iburst'", retry => 5, delay => 30);
```
We removed the `sleep 5`. Let's make the test run try 5 times before giving up to avoid that openQA runs too fast.